ww-config.js

```diff
@@ -12,6 +12,15 @@ export default {
         {
             name: 'dateISO',
             arrity: 0,
+            deprecated: true,
+        },
+        {
+            name: 'date',
+            arrity: 0,
```

Comment:
@AlexisJC  Curious, what means "arrity"?

Reply:
arrity = number of arguments